.github/workflows/test.yml

```diff
@@ -21,6 +21,7 @@ env:
   VERBOSE: true
   CI: true
   PROFILE: debug
+  RUSTFLAGS: "-D warnings"
```

Comment:
I think it would be great to link the [doc](https://doc.rust-lang.org/rustc/lints/groups.html) in the PR description, so it is easier for a next reader to understand what is flag mean.

Reply:
Fixed